### ISSUE TITLE
Local installation error: Failed at the cloudgov-style@0.3.4 watch script 'watch 'npm run build' ./src'.

### ISSUE THREAD

**Issue by mrengy** [18F/cg-landing#62]
Hi there. I'm trying to get this site running locally so that I can contribute to it. I followed the [installation instructions](https://github.com/18F/cg-landing#installation), but I'm running into an error.
1. I cloned this repository as well as the cg-style repository.
2. I ran `gem install github-pages`. That showed messages of: `Successfully installed github-pages-69` and `Done installing documentation for jekyll-github-metadata, github-pages after 1 seconds
   2 gems installed`.
3. I ran `jekyll serve --baseurl='' -w`. That allowed me to view the site locally, but without any styling applied to it.
4. In my local directory for cg-style, I ran `npm run watch` and got the following error:

> Failed at the cloudgov-style@0.3.4 watch script 'watch 'npm run build' ./src'.
> npm ERR! This is most likely a problem with the cloudgov-style package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     watch 'npm run build' ./src

Here's the [entire output from npm-debug.log](https://gist.github.com/mrengy/93967071b123ea5016b6807b1fb4cc57).

I'm on Mac OS 10.6, node v4.4.3, npm v2.15.1

Thanks for any assitance.

**Comment by msecret** [comment]
@mrengy quick question: do you want to change the styling of the landing page at all? If you don't, you don't need to install and configure cg-style at all.

I'm in the process now of updating the documentation for installing cg-style.

**Comment by mrengy** [comment]
Yes, this would involve that styling, I believe. I was going to take a crack at resolving this issue: https://github.com/18F/cg-landing/issues/43

**Comment by msecret** [comment]
@mrengy OK I opened a PR with updated setup instructions: https://github.com/18F/cg-landing/pull/63

Maybe try that out and see if it works?

**Comment by mrengy** [comment]
Thanks for the quick reply. I followed the instructions at that [updated Readme file](https://github.com/18F/cg-landing/blob/ms-update_docs/README.md).
1. Ran `gem install github-pages`
2. Ran `jekyll serve --baseurl='' -w`
3. I already have the cg-style repository cloned.
4. From the cg-style directory, I ran `npm install cloudgov-style --save` and then got a bunch of errors.

Here's the [output from the Terminal](https://gist.github.com/mrengy/10bc099471b3541ab1d6e2fae1c70084).

And the [full npm-debug.log file](https://gist.github.com/mrengy/e5aca026b266b17f48cff4b2cdc126ce).
